pkg/controller/node/controller.go

```diff
@@ -65,6 +72,9 @@ func New(
 		nodeClassStore:    nodeClassStore,
 		mapi:              mapi,
 		client:            client,
+		nodes:             nodes,
+		syncErrors:        syncErrors,
+		syncSeconds:       syncSeconds,
```

Comment:
For simplicity i would rather put them into a separate package with exported functions to interact